app/validators/admin/facility_csv_validator.rb

```diff
@@ -0,0 +1,48 @@
+class Admin::FacilityCSVValidator
```

Comment:
Feel like I prefer `Admin::CSV::FacilityValidator` over `FacilityCSVValidator`.

Reply:
Addressed in dc5d307.